### ISSUE TITLE
Only one user can have a custom git integration

### ISSUE THREAD

**Issue by ghost** [gitpod-io/gitpod#5411]
### Bug description

I just showed off the product and how it works for another colleague and it seems like only one user can integrate to a custom gitlab server. I had to remove it from mine so that he could do the integration, but as you see in the picture i am no longer able to integrate even tough i have no integrations setup on my personal account

![Screenshot 2021-08-27 at 10 08 30](https://user-images.githubusercontent.com/53090349/131095688-74175208-f9d5-4e01-80c1-4c4af41320c7.png)

### Steps to reproduce

Try to add a custom git integration with "ic-dev-03.increo.space" as the "Provider Host Name", the rest does not matter if it is valid or not

### Expected behavior

Integrations to be connected per user and not globally, does this mean that other users could access the integration towards "ic-dev-03.increo.space" seeing as it says "Provider for this host already exists."

**Comment by RobinHoutevelts** [comment]
I have a gitlab instance I share with friends.

I also have a paid gitpod account but most friends don't. If I add the integration, and my friends connect with their own account does it share data/workspaces with my friends?

- Are they going to be able to create workspaces by crafting an url?
- Are those in any way going to be billed to me?
- Are they going be able to create workspaces for projects they don't have access to?

---

It seems to me I want them to add their own damn integration. Feels icky that they would use my integration

**Comment by AjXUthaya** [comment]
I don't think this as intuitive to have to start a project to be able to integrate for a new user, i nor my colleague though this so at the very least there should some more information instead of "Provider for this host alread exists". That just seems like an error from my point of view where i have to disconnect for someone else to be able to connect their account.

Seems like this product aims to make it as simple as possible for the end users, and i think some more information would help other end users reach this goal instead of filing an bug report when it is not a bug but a feature :)

**Comment by JanKoehnlein** [comment]
I think the error message should convey 
1) That another integration is not necessary (maybe with a link to the docs)
2) What users have to do to open a repo on that git hoster, e.g. link to "New project" or "New workspace"

I am not sure why we should expose the handle of the integrator, as their is o use in contacting them.

We should also clarify the role of the "integrator" and what an integration does in the docs.
